applications/prompt-keda-hsc/values.yaml

```diff
@@ -67,6 +67,10 @@ prompt-keda:
   # If empty, Prompt Processing does not use a microservice.
   raw_microservice: ""
 
+  # -- The URI to the MPSky ephemerides service.
+  # Empty value is allowed, but its handling is undefined.
+  mpSky_service: ""
```

Comment:
Copying @Gerenjie's comments from Jira:
> Wouldn’t the current setting, `mpSky_service: ""`, pass the empty string (rather than no value) to the environment variable, spoiling the intended effect of undefined value rather than empty value? 
>
> For example, when I set the MP_SKY_URL environment variable to '', I get the wrong effect of os.environ.get('MP_SKY_URL', fallback) returning '' rather than fallback.

Reply:
The `with` action (I think that's the term?) in the template takes care of that: if `mpSky_service` is non-empty (bool conversion works the same way in Go templates as it does in Python), it defines `MP_SKY_URL` with the same value. If it _is_ empty, it doesn't define `MP_SKY_URL` at all. That keeps the user-facing YAML config from being hypersensitive to exactly whether it needs to be `""`, `null`, etc.